docker-compose-setup/docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+services:
+  coatrack-postgres:
+    image: postgres:9.4.26
+    container_name: coatrack-postgres
+    volumes:
+      - coatrack-postgres:/var/lib/postgresql/data
+    environment:
+      POSTGRES_PASSWORD: password1234
+  coatrack-config-server:
+    image: coatrack/coatrack-config-server:2.0.0-SNAPSHOT
+    container_name: coatrack-config-server
+    depends_on:
+      - coatrack-postgres
+    ports:
+      - 8998:8998
+    entrypoint: ["sh", "-c", "java -Djava.security.egd=file:/dev/./urandom \
+      -jar /opt/coatrack/lib/coatrack-config-server.jar \
+      --spring.datasource.url=jdbc:postgresql://coatrack-postgres/ygg_config_server \
+      --spring.datasource.password=password1234"]
+  coatrack-admin:
+    image: coatrack/coatrack-admin:2.0.0-SNAPSHOT
+    container_name: coatrack-admin
+    volumes:
+      - coatrack-admin:/home
+    depends_on:
+      - coatrack-config-server
+      - coatrack-postgres
+    ports:
+      - 8080:8080
+    entrypoint: [ "sh", "-c", "sleep 5 && java -Djava.security.egd=file:/dev/./urandom \
+      -jar /opt/coatrack/lib/coatrack-admin.jar \
```

Comment:
Here should then be added the email config, or it will get an error while running

Reply:
This is strange. I repeated the instruction in the PR description and didn't encounter this problem.

Reply:
This seems to be a more general issue, not related to this specific PR, see also: https://github.com/coatrack/coatrack/pull/164#issuecomment-975341873

Reply:
This issue is already solved